Profiles/cio-Symptom.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="cio-Symptom" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/cio-Symptom" />
+  <version value="1.0.0-beta.3" />
+  <name value="CioSymptom" />
+  <title value="cio Symptom" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A symptom is a phenomenon with which a Observation in the patient presents itself. This also includes a complaint or a finding, but not additional research, such as laboratory tests, function tests and imaging studies." />
+  <purpose value="This Observation resource represents the Symptom related concepts for implementations following the information standard [CiO](https://informatiestandaarden.nictiz.nl/wiki/Landingspagina_Contra-indicaties_en_Overgevoeligheden). This profile is largely based on the Dutch zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) Symptom is part of the 2024 prepublication. Therefore it has no dependency on a corresponding nl-core profile and mappings to zib concepts are not defined. Instead mappings to the CiO data set have been added. It is the intention to derive this profile from a more general nl-core-Symptom profile in the future." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="cio-dataset-200-beta2-20231214" />
+    <uri value="https://decor.nictiz.nl/pub/cio/cio-html-20231214T100742/ds-2.16.840.1.113883.2.4.3.11.60.26.1.2-2020-04-02T000000.html" />
+    <name value="ART-DECOR Dataset Contraindications and hypersensitivities 2.0.0-beta.2 20231214" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Observation" />
```

Comment:
I'm still contemplating whether Observation is the right resource to map a Symptom on, especially due to concepts such as DurationWhenPresent. On the other hand, the FHIR specification states the following:

_This resource is not typically used to record information about subjective and objective information that might lead to the recording of a Condition resource. Such signs and symptoms are typically captured using the [Observation](http://hl7.org/fhir/R4/observation.html) resource; although in some cases a persistent symptom, e.g. fever, headache may be captured as a condition before a definitive diagnosis can be discerned by a clinician._

So in seems that Observation might be the correct resource, since there is always an accompanying Condition resource (corresponding to the Condition building block).

Short version: Observation seems the way to go!

Reply:
If the Symptom zib would be mapped to a Condition profile in nl-core, it's better to already implement it that way in CiO as well (even though elements such as Beloop are not that relevant for CiO (yet)), since that reduces rework in the future and is also a better 'message' to implementers.

Reply:
I'm not certain about the nl-core profile being mapped to Condition. I was not able to find any analysis done by the nl-core team